packages/design-system-docs/src/scripts/components/PageBlock.jsx

```diff
@@ -63,33 +91,30 @@ class PageBlock extends React.PureComponent {
     // text below the title + code snippet block. It's hacky, but works.
     if (this.props.hideHeader || this.props.header.match(/---/)) return;
 
-    const source = this.props.reactComponentPath && (
-      <Source key="reactSource" reactComponentPath={this.props.reactComponentPath} />
-    );
+    const subheader = this.props.reactComponentPath && <span key="subheader">React&nbsp;</span>;
```

Comment:
Personally I'm not a big fan of adding the "React component" wording. I think it will be a bit clunky and repetitive looking when there are multiple, like in `Tab` or `ChoiceList`. I would prefer to bring back the tabs we used to have to separate into html, react, and guidance sections. Of course this might not be a big priority right now, and I think ideally we have a designer look at our design in general, so I would prefer to keep the UI here same if possible